keyboards/plaid/keymaps/tominabox1/keymap.c

```diff
@@ -0,0 +1,103 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    /* Colemak
+     * ,-----------------------------------------------------------------------------------.
+     * | Q    |   W  |   F  |   P  |   G  |  ESC |   DEL|   J  |   L  |   U  |   Y  | :    |
+     * |------+------+------+------+------+-------------+------+------+------+------+------|
+     * | A    |   R  |   S  |   T  |   D  |  TAB |RETURN|   H  |   N  |   E  | I    |  O   |
+     * |------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Z    |   X  |   C  |   V  |   B  |   [  |   ]  |   K  |   M  |   ,  |   .  |   '  |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * | CTRL | CAD  | GUI  | BSPC | LALT | TG3  | MO3  | SHIFT| SPACE|RETURN| BSLS | UP   |
+     * `-----------------------------------------------------------------------------------'
+     */
+    
+    [0] = LAYOUT_ortho_4x12(
+        KC_Q, KC_W, KC_F, KC_P, KC_G, KC_GESC, KC_DEL,  KC_J, KC_L, KC_U,    KC_Y,   KC_SCLN, \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

Reply:
I will remove these as requested in the next update.